README.md

```diff
@@ -80,6 +81,12 @@ format>
 
 Where:
 
+--response-header-with-latency-input <string>
+Set an optional response header name, whose values will be tracked in
+a latency histogram if set. Can be used in tandem with the test server
```

Comment:
For the second sentence, can we edit to something like:
"... if set.  e.g. Can be used in tandem with the test server's response option "emit_previous_request_delta_in_response_header" to record elapsed time between request arrivals."

(Adding in the word response option clarifies where the reader should look for the definition of that)